debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -691,6 +694,17 @@ public static FieldNameAdjustmentMode parse(String value) {
             .withImportance(Importance.LOW)
             .withDescription("The name of the SourceInfoStructMaker class that returns SourceInfo schema and struct.");
 
+    public static final Field MAX_RETRIES_ON_ERROR = Field.create(ERRORS_MAX_RETRIES)
+            .withDisplayName("The maximum number of retries")
+            .withType(Type.INT)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR_ADVANCED, 2))
```

Comment:
2nd position in the group is already occupied by `ROW_COUNT_FOR_STREAMING_RESULT_SETS` in Mysql connector config (this is also why Mysql tests fail). You need to use some unused position. If I'm not mistaken, the next one which is free is 22.

Reply:
@vjuranek 
i fix the group number
is there any way to manage this group number? 
the group numbers appear out of order.


![스크린샷 2023-06-29 오후 5 12 47](https://github.com/debezium/debezium/assets/90180644/c8a904e5-5022-434f-a85b-7c74c7c91322)

Reply:
@jaegwonseo I'm afraid we haven't any way how to manage these numbers:-(
Could you please also sqush the commits?